docs/content/any/project/changelogs/2021-11-09.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Release 2021-11-09
+menuTitle: "2021-11-09"
+any: true
+description: >-
+  Changelog for Release 2021-11-09 (0.23.0) containing new features,
```

Comment:
name of file & every reference in here should be `2021-11-10` (tomorrow) instead of the 9th (today)